doc/source/pbm-commands.rst

```diff
@@ -93,13 +170,50 @@ The command accepts the following flags:
      - Description
    * - ``--time=TIME``
      - Restores the database to the specified point in time. Available if :ref:`PITR` is enabled.
+   * - ``-o``, ``--out=text``
+     - Shows the output format as either plain text or a JSON object. Supported values: text, json
+       
+.. container:: toggle
+
+   .. container:: header
+
+      **JSON output**
+
+   .. code-block:: javascript
+
+      {
+        "point-in-time": "<backup_name>"
+      }
```

Comment:
this is output for `pbm restore --time=...`
for `pbm restore ...` (snapshot restore, NOT pitr) it would be slightly different:
```
{"snapshot":"2021-08-09T14:21:40Z"}
```

Reply:
yep, I left an extended version as an example.

Reply:
Added